autoscaler.go

```diff
@@ -28,13 +28,17 @@ type Autoscaler struct {
 func (a *Autoscaler) Start() {
 	for range time.Tick(a.Interval) {
 		log.Println("Calculating scales for all")
-		// TODO: Might need to account for queue size being 100
-		// probably can simply divide the average by queue size
 		for deploymentName, waitCount := range a.FIFO.WaitCounts() {
 			avg := a.getMovingAvgQueueSize(deploymentName)
 			avg.Next(float64(waitCount))
 			flt := avg.Calculate()
-			ceil := math.Ceil(flt)
+			// TODO fix this to use configurable concurrency setting that's supplied
```

Comment:
I think this TODO comment can be removed.

Reply:
I rather keep it for myself since its not obvious what to change. I might forget in a few days when i try to make concurrency configurable